configurator/src/capture.rs

```diff
@@ -61,8 +61,17 @@ impl Capturer {
 
     pub async fn capture_perspective(
         &mut self,
+        save_pictures: bool,
     ) -> Result<Vec<Option<(usize, usize)>>, Box<dyn Error>> {
         let mut coords = Vec::new();
+        let timestamp = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)?
+            .as_millis()
```

Comment:
Wouldn't it be more readable in the ISO format? Year, month, day, hour, minutes, seconds. Also not sure if we need it down to millis, it's not like you'll be able to capture more than 1 in one second.

Reply:
Done